packages/ui/src/themes/index.ts

```diff
@@ -1,6 +1,6 @@
-import { createTheme } from '@mui/material';
+import { createTheme, PaletteMode } from '@mui/material';
 
-export default createTheme({
+const theme = (mode: PaletteMode) => createTheme({
```

Comment:
Let's rename the `theme` function to `newTheme` to make it more clear.
```suggestion
const newTheme = (mode: PaletteMode) => createTheme({
```